From e77e5783da0d2ecf7def4c403069979f00d72559 Mon Sep 17 00:00:00 2001
From: spatil1985 <sudip.patil20@gmail.com>
Date: Sat, 20 Jan 2018 13:36:31 +0530
Subject: [PATCH 1/7] 4

---
 GitTutorial/common.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/GitTutorial/common.txt b/GitTutorial/common.txt
index a51d979..9988069 100644
--- a/GitTutorial/common.txt
+++ b/GitTutorial/common.txt
@@ -1 +1 @@
-Branch2
\ No newline at end of file
+Branch1
\ No newline at end of file
-- 
2.16.0.windows.2

