From 1d1e01f963528cd385a6ccfec6135ec8f9ba91a3 Mon Sep 17 00:00:00 2001
From: spatil1985 <sudip.patil20@gmail.com>
Date: Sat, 20 Jan 2018 16:53:30 +0530
Subject: [PATCH 2/7] 3

---
 GitTutorial/common.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/GitTutorial/common.txt b/GitTutorial/common.txt
index 9988069..55b7a3a 100644
--- a/GitTutorial/common.txt
+++ b/GitTutorial/common.txt
@@ -1 +1 @@
-Branch1
\ No newline at end of file
+Master Changes
-- 
2.16.0.windows.2

