From f18e920f10a5b135897a405cdff153a4c2ed99b7 Mon Sep 17 00:00:00 2001
From: spatil1985 <sudip.patil20@gmail.com>
Date: Sat, 20 Jan 2018 16:55:12 +0530
Subject: [PATCH 3/7] 3

---
 GitTutorial/common.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/GitTutorial/common.txt b/GitTutorial/common.txt
index 9988069..059a4e0 100644
--- a/GitTutorial/common.txt
+++ b/GitTutorial/common.txt
@@ -1 +1 @@
-Branch1
\ No newline at end of file
+Branch 1 Changes
-- 
2.16.0.windows.2

